pkgs/applications/networking/browsers/ladybird/default.nix

```diff
@@ -8,25 +8,26 @@
 , makeWrapper
 , qtbase
 , qttools
+, nixosTests
 }:
 
 let serenity = fetchFromGitHub {
   owner = "SerenityOS";
   repo = "serenity";
-  rev = "094ba6525f0217f3b8d5e467cef326caeb659e8a";
-  hash = "sha256-IHXe2Td9iRSL1oQVwL2gZHxEM2ID4SghZwK6ewjFV1Y=";
+  rev = "ae8f1c7dc88e5bd79fb3e232e540ddc3dd2f1c11";
+  hash = "sha256-1OcSBwEs/SGocTlOoVEv+2bTg4kqtUT2TUBOWG7BqkE=";
 };
 
 in gcc11Stdenv.mkDerivation {
   pname = "ladybird";
-  version = "unstable-2022-07-20";
+  version = "unstable-2022-08-14";
 
   # Remember to update `serenity` too!
   src = fetchFromGitHub {
     owner = "awesomekling";
     repo = "ladybird";
-    rev = "9e3a1f47d484cee6f23c4dae6c51750af155a8fc";
-    hash = "sha256-1cPWpPvjM/VcVUEf2k+MvGvTgZ3Fc4LFHZCLh1wU78Y=";
+    rev = "35a6f69d65fcdb17fb9f84247fe8caf2b49f7c7d";
```

Comment:
FYI, upstream issue 36 referenced in the installPhase and preFixup steps was addressed (hopefully in a way that jives with nixpkgs) in <https://github.com/SerenityOS/ladybird/commit/7c859618e097cffceee67c170779a6b03d3837c0>

It should also eliminate the need to pass SERENITY_SOURCE_DIR, as the app will search in <app location>/../share/res for resources instead of expecting the env var to point somewhere useful.

Reply:
Thanks for the heads up, I'll clean up the package on the next update

Reply:
Please do it right now so that it does not get forgotten.

Reply:
Done! I also addded darwin to platforms since it's supposed to build there too (though currently it doesn't because we don't have qt6 darwin builds).

That was a nice cleanup, the .nix is a lot smaller now!

![screenshot](https://user-images.githubusercontent.com/8182846/193227602-9367b64d-4890-4c9a-8926-3d80c1681129.png)